supporting_content/use_cases/compare_billed_cost_per_subaccount_to_budget.md

```diff
@@ -0,0 +1,54 @@
+# Use Case
+
+Compare billed cost per subaccount to budget
```

Comment:
@beau-nelford-anglepoint What changed in 1.1 that impacted this use case? None of the columns here are flagged as changed in the 1.1 changelog.